client-ts/signalr/src/third-party-notices.txt

```diff
@@ -0,0 +1,32 @@
+.NET Core uses third-party libraries or other resources that may be
```

Comment:
Also paging @Eilon (just to make sure this looks right)

Reply:
Looks good.